index.html

```diff
@@ -3097,6 +3097,20 @@ <h3>Production</h3>
           </li>
         </ul>
 
+        <p>
+          To produce a deterministic canonical CBOR representation of a DID document and faciliate maximal lossless compatiblity with other 
+          core representations via the Abstract Data Model the following constraints of a CBOR representation of a DID Document model MUST be followed:
+          
```

Comment:
typos and line lengths
```suggestion
To produce a deterministic canonical CBOR representation of a DID
document and facilitate maximal lossless compatibility with other core
representations via the Abstract Data Model the following constraints
of a CBOR representation of a DID Document model MUST be followed:
        </p>
```